src/interfaces/api/community.ts

```diff
@@ -118,3 +119,17 @@ export interface TransferCommunity {
   person_id: number;
   auth: string;
 }
+
+export interface BlockCommunity {
+  community_id: number;
+  block: boolean;
+  auth: string;
+}
+
+export interface GetBlockedCommunities {
```

Comment:
Should this be in `src/interfaces/api/person.ts` since it is under `GET /user/blocked_communities`?